### ISSUE TITLE
rx.d.ts compile error

### ISSUE THREAD

**Issue by christyharagan** [Reactive-Extensions/RxJS#955]
As of v4, the rx.d.ts file ends like:

```TypeScript
declare module "rx" { export = Rx; }
declare module "rx" { export = Rx; }
```

This should not be a double entry as it causes the typescript compiler to complain of duplicate entries.

**Comment by shadeglare** [comment]
Have the same issue.

**Comment by mattpodwysocki** [comment]
@david-driscoll any idea on this?

**Comment by baltie** [comment]
Fixed by PR #961.
